utils/src/responses/UserResponse.ts

```diff
@@ -0,0 +1,56 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { State, Role } from '../enums';
+
+export class UserResponse {
+  @ApiProperty({
+    description: 'User\'s id',
+  })
+    id: string;
+
+  @ApiProperty({
+    description: 'User\'s email',
+  })
+    email: string;
+
+  @ApiProperty({
+    description: 'User\'s FICE SC Platform username',
```

Comment:
There is no need to write project's name in field description
```suggestion
    description: 'User\'s username',
```